far/vmenu.cpp

```diff
@@ -181,9 +181,28 @@ struct menu_layout
 	}
 };
 
+enum class item_hscroll_policy
+{
+	unbound,            // The item can move freely beyond window edges.
+	cling_to_edge,      // The item can move beyond window edges, but at least one character is always visible.
+	bound,              // The item can move only within the window boundaries.
+	bound_stick_to_left // Like bound, but if the item shorter than TextAreaWidth, it is always attached to the left window edge.
+};
+
 namespace
 {
-	MenuItemEx far_list_to_menu_item(const FarListItem& FItem)
+	const FarListItem string_to_far_list_item(const wchar_t* NewStrItem)
```

Comment:
Remove const from the return value please.

Reply:
Thank you for catching it.